.github/workflows/integrate-and-deploy.yml

```diff
@@ -14,6 +14,9 @@ on:
       image-tag-next-app:
         type: string
         required: true
+      image-tag-lfmerge:
```

Comment:
the lfmerge version is really driven from the `.json`, I'm not sure we actually need this at all.  Was there a use-case you were imagining where we'd tweak the `staging.yml` or `production.yml` to feed this workflow a version?

Reply:
I think I was picturing a use case where we'd add a workflow_dispatch trigger to `integrate-and-deploy.yml`, in which case it would be useful to have the ability to override the .json values. Say, because we've discovered that I introduced a bug in the current version of LfMerge and want to quickly roll back to the previous version.

You're right that that's not needed most of the time, but I thought having an override available might be a good idea.

Reply:
let's wait until that need comes up.

Reply:
Done in commit d7b3e50.